CONTRIBUTING.md

```diff
@@ -13,6 +13,20 @@ Run `python setup.py sdist`
 
 Use `pip install -e <path>` to install it into test application for the testing purposes.
 
+## Generate documentation
+
+1. Install [sphinx](http://www.sphinx-doc.org/en/master/usage/installation.html): `brew install sphinx-doc`.
```

Comment:
brew is Macintosh specific 👎 .

The cross-platform method is to make a virtualenv and install sphinx into it:
```bash
% python3 -m venv env       # Create 'env' directory
% . env/bin/activate        # Activate virtual environment
% pip install sphinx django # Install sphinx and django (so sphinx can find it)
% python setup.py install   # Install applicationinsights package
% cd doc
% make html
.
.
.
% deactivate   # Exit virtualenv
```